docs/releases/v3.0.3.md

```diff
@@ -0,0 +1,37 @@
+# Monitoring Core Module Release 3.0.3
+
+Welcome to the latest release of the `monitoring` core module of the [`Kubernetes Fury Distribution`](https://github.com/sighupio/fury-distribution), maintained by team SIGHUP.
+
+This update includes support for Kubernetes 1.28-1.29 and upgrades some of the components
+
+## Component Images 🚢
+
+| Component             | Supported Version                                                                                   | Previous Version |
+| --------------------- | --------------------------------------------------------------------------------------------------- | ---------------- |
+| `alertmanager`        | [`v0.26.0`](https://github.com/prometheus/alertmanager/releases/tag/v0.26.0)                        | No Update        |
+| `blackbox-exporter`   | [`v0.24.0`](https://github.com/prometheus/blackbox_exporter/releases/tag/v0.23.0)                   | No Update        |
+| `grafana`             | [`v9.5.5`](https://github.com/grafana/grafana/releases/tag/v9.5.5)                                  | No Update        |
+| `kube-rbac-proxy`     | [`v0.14.0`](https://github.com/brancz/kube-rbac-proxy/releases/tag/v0.14.0)                         | No Update        |
+| `kube-state-metrics`  | [`v2.9.2`](https://github.com/kubernetes/kube-state-metrics/releases/tag/v2.9.2)                    | No Update        |
+| `node-exporter`       | [`v1.6.1`](https://github.com/prometheus/node_exporter/releases/tag/v1.6.1)                         | No Update        |
+| `prometheus-adapter`  | [`v0.11.1`](https://github.com/kubernetes-sigs/prometheus-adapter/releases/tag/v0.11.1)             | No Update        |
+| `prometheus-operator` | [`v0.67.1`](https://github.com/prometheus-operator/prometheus-operator/releases/tag/v0.67.1)        | No Update        |
+| `prometheus`          | [`v2.46.0`](https://github.com/prometheus/prometheus/releases/tag/v2.41.0)                          | No Update        |
+| `thanos`              | [`v0.34.0`](https://github.com/thanos-io/thanos/releases/tag/v0.34.0)                               | `v0.32.2`        |
+| `x509-exporter`       | [`v3.12.0`](https://github.com/enix/x509-certificate-exporter/releases/tag/v3.12.0)                 | `v3.6.0`         |
+| `karma`               | [`v0.113`](https://github.com/prymitive/karma/releases/tag/v0.113)                                  | No Update        |
+| `mimir`               | [`v2.11.0`](https://github.com/grafana/mimir/releases/tag/mimir-2.11.0)                             | `v2.10.3`        |
+| `minio-ha`            | [`RELEASE.2024-02-09T21-25-16Z`](https://github.com/minio/minio/tree/RELEASE.2024-02-09T21-25-16Z)  | `RELEASE.2023-01-12T02-06-16Z` |
+
+> Please refer the individual release notes to get a detailed info on the releases.
+
+## Update Guide 🦮
+
+### Process
+
+To upgrade this core module from `v3.0.1` to `v3.0.2`, execute the following:
```

Comment:
this is referencing the previous release

Reply:
Solved by 5fe5cc04a6ff19047cb4591b14ad4f3a5e0149cc